terraform/variables.tf

```diff
@@ -27,4 +27,10 @@ locals {
     cidr_subnet1_db      = "192.168.4.0/24"
     cidr_subnet2_db      = "192.168.5.0/24"
   }
+
+  newvpc = {
```

Comment:
Adding new subnets, Needed to change the masking to /24 since /16 creates overlaps for this type of IP address